packages/block-library/src/section/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	InnerBlocks,
+	PanelColorSettings,
+	withColors,
+} from '@wordpress/editor';
```

Comment:
I think we should use `block-editor` here. These are proxified now.

Reply:
✅ Done